packages/twenty-front/src/modules/ui/layout/modal/components/__stories__/ModalLayout.stories.tsx

```diff
@@ -1,31 +1,31 @@
 import { Meta, StoryObj } from '@storybook/react';
 import { ComponentDecorator } from 'twenty-ui';
 
-import { ModalLayout } from '@/ui/layout/modal/components/ModalLayout';
+import { EnhancedModalLayout } from '@/ui/layout/modal/components/EnhancedModalLayout';
 
-const meta: Meta<typeof ModalLayout> = {
+const meta: Meta<typeof EnhancedModalLayout> = {
   title: 'UI/Layout/Modal/ModalLayout',
-  component: ModalLayout,
+  component: EnhancedModalLayout,
 };
 
 export default meta;
-type Story = StoryObj<typeof ModalLayout>;
+type Story = StoryObj<typeof EnhancedModalLayout>;
```

Comment:
Oh so we deprecate ModalLayout? We not leave it named as ModalLayout and leave it like this then?

I saw in your nice PR you commented some code. Don't be afraid to delete files and code, that's what Git is for, we can always revert if it wasn't the right strategy!

Reply:
yep we decpreciate ModalLayout and Modal(which is used for hotscoped)

Reply:
Sorry I misunderstood!

Reply:
Yes I agree, we haven't finished migrating components to twenty-ui! It's on the roadmap but I didn't really follow this topic closely